src/modules/acbs/known-errors.ts

```diff
@@ -21,7 +21,7 @@ export const getDealNotFoundKnownAcbsError = (dealIdentifier: string): KnownErro
 });
 
 export const getFacilityNotFoundKnownAcbsError = (facilityIdentifier: string): KnownError => ({
-  substringToFind: 'The facility not found',
+  substringToFind: 'acility not found', // TODO: is this 'The facility not found' or 'Facility not found', or can it be either?
```

Comment:
I can replicate both error messages:
- POST facility guarantee gives - The facility not found or the user does not have access to it.
- POST facility covenant gives - Facility not found or the user does not have access to it.

Happy to use short string "acility not found" , but lets leave comment why it was used.

Reply:
Left a comment: "This is used because ACBS uses the wording 'Facility not found...' for some endpoints and 'The facility not found...' for others."